keyboards/fungo/rev1/rules.mk

```diff
@@ -0,0 +1,32 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu   # controller Elite-C
```

Comment:
Is this using pins that are not available to the pro micro board?

Reply:
I don't know what you want to know.
It can use pro-micro, nicenano, and elite-c boards. I am currently using the elite-c board, so I set it up as above.
What is wrong?

Reply:
For boards that use the Pro Micro style layout, we generally want to use the caterina bootloader, as that's assumed to be the default. 

You can specify a different bootloader in the case of an elite c, or the like. 

This is especially the case, as Pro Micros don't have a hardware reset button on the controller, but boards like the elite C do.  So if the wrong bootloader is set on a pro micro.... you're going to be SOL getting into the bootloader to reflash the board, without exposing the controller and shorting the RST pin to GND.   In fact, this is one of the reasons we recommend setting it to caterina by default (the pro micro's default bootloader).

Reply:
I know that.
So how should I modify the code?
My previous code is as below.But I deleted it because you told me to modify it.What should I do now? Let me know exactly.

BOOTLOADER = atmel-dfu   # controller Elite-C
# BOOTLOADER = caterina   # controller pro micro

Reply:
Ideally, it should be just left as caterina.